packages/string-templates/src/helpers/javascript.js

```diff
@@ -56,10 +56,12 @@ module.exports.processJS = (handlebars, context) => {
     const res = { data: runJS(js, sandboxContext) }
     return `{{${LITERAL_MARKER} js_result-${JSON.stringify(res)}}}`
   } catch (error) {
-    console.log(`JS error: ${typeof error} ${JSON.stringify(error)}`)
     if (error.code === "ERR_SCRIPT_EXECUTION_TIMEOUT") {
       return "Timed out while executing JS"
     }
+    if (error.name === "ExecutionTimeoutError") {
```

Comment:
```suggestion
    if (error.name === ExecutionTimeoutError.name) {
```

Reply:
Removing some magic strings

Reply:
I think doing this would require `string-templates` to have a dependency on `backend-core`, which I suspect we don't want because this gets used in the frontend as well.

Reply:
Does this mean that we are adding reference `string-templates` that should not be part of it? Would it not be a better practice to bubble up the error instead of trying to cast the types?

Reply:
It would almost certainly be better to bubble up the error, yes. I was sticking to how it works today (we already catch all errors and return a string here, I don't know the history behind it).

I would prefer to come back to this at a later date though if that's alright. It's an existing problem that we're not making much worse here, and I have a feeling it'll be a tricky change.

Reply:
That's ok, we should revisit this library if we keep extending it. `string-templates` is usually pretty static